exchange/c2cx/api.go

```diff
@@ -481,6 +482,34 @@ func (c *Client) MarketSell(symbol TradePair, amount decimal.Decimal, customerID
 	return orderID, nil
 }
 
+type GetTickerResponse struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+	Data    struct {
+		Timestamp string  `json:"timestamp,omitempty"`
```

Comment:
Make this a `time.Time` and implement custom `UnmarshalJSON` and `MarshalJSON` to handle it.  See examples in `c2cx.go`.